site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -24,17 +24,79 @@ permission in the manifest, as shown below:
 
 ## Examples
 
-![Two tabs in a window](tabs.png)
+The following sections demonstrate several common use cases for the chrome.tabs API.
 
-You can find simple examples of manipulating tabs with the `chrome.tabs` API in the
-[examples/api/tabs][7] directory. For other examples and for help in viewing the source code, see
-[Samples][8].
+### Opening an extension page in a new tab
 
-[1]: /docs/extensions/mv2/tabs
-[2]: #property-Tab-url
-[3]: #property-Tab-pendingUrl
-[4]: #property-Tab-title
-[5]: #property-Tab-favIconUrl
-[6]: #type-Tab
-[7]: https://github.com/GoogleChrome/chrome-extensions-samples/tree/master/mv2-archive/api/tabs/
-[8]: /docs/extensions/mv2/samples
+A common pattern for extensions is to open an onboarding page in a new tab when the extension is
+installed. The following example shows how to do this.
+
+{% Aside %}
+Cannot use `chrome.tabs.create()` in content scripts.
+{% endAside %}.
+
+```js
+//// background.js
+
+chrome.runtime.onInstalled.addListener((reason) => {
+  if (reason === chrome.runtime.OnInstalledReason.INSTALL) {
+    chrome.tabs.create({
+      url: 'onboarding.html'
+    });
+  }
+});
+```
+
+### Get the current tab
+
+This example demonstrates how the background script can retrieve the currently focused tab.
+
+{% Aside %}
+Requires Manifest V3 due to the use of Promises. Cannot use `tabs.query` in content
+scripts.
+{% endAside %}
+
+```js
+//// background.js
+
+async function getCurrentTab() {
+  let queryOptions = { active: true, currentWindow: true };
+  let [tab] = await chrome.tabs.query(queryOptions);
+  return tab;
+}
+```
+
+### Mute the specified tab
+
+This example shows how an extension can toggle the muted state for a given tab.
+
+{% Aside %}
+Requires Manifest V3 due to the use of Promises. Cannot use `tabs.get` or `tabs.update`
+in content scripts.
+{% endAside %}
+
+```js
+//// background.js
+
+function toggleMuteState(tabId) {
+  chrome.tabs.get(tabId, async (tab) => {
+    let muted = !tab.mutedInfo.muted;
+    await chrome.tabs.update(tabId, { muted });
+    console.log(`Tab ${tab.id} is ${ muted ? 'muted' : 'unmuted' }`);
+  });
+}
+```
```

Comment:
> I'd like to frame this as a use case; is there an obvious one? 

FYI @awfuchs, I replaced the navigation request example with this new example and opened #665 to revisit a message passing example in the future.